### PR TITLE
Some code review

### PR DESCRIPTION
**Pull request by HuKeping**
It's a good choice to use the constants we already have.

Signed-off-by: Hu Keping <hukeping@huawei.com>

### CODE REVIEW

File: server/handlers/changefeed.go [hunk 1]
```diff
@@ -63,7 +63,7 @@ func checkChangefeedInputs(logger ctxu.Logger, s interface{}, r string) (
 
 	store, ok := s.(storage.MetaStore)
 	if !ok {
-		logger.Error("500 GET unable to retrieve storage")
+		logger.Errorf("%d GET unable to retrieve storage", http.StatusInternalServerError)
 		err = errors.ErrNoStorage.WithDetail(nil)
 		return
 	}
```
Comment (riyazdf) [inline server/handlers/changefeed.go]: nit: could you also update the `logger.Errorf("400...` below to use `http. StatusBadRequest`?

**Comment by endophage** [comment]
LGTM pending Jenkins working out its problems

File: server/handlers/changefeed.go (hunk 1 above)
Comment (HuKeping) [inline server/handlers/changefeed.go]: Thanks @riyazdf I'll update

**Comment by cyli** [comment]
Thanks @HuKeping!!  :)

**Pull request merged**